src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -148,6 +148,14 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):  # pylin
     register_cli_argument(scope, 'no_auto_upgrade', action='store_true', help='by doing this, extension system will not pick the highest minor version for the specified version number, and will not auto update to the latest build/revision number on any scale set updates in future.')
     register_cli_argument('{} create'.format(scope), 'generate_ssh_keys', action='store_true', help='Generate SSH public and private key files if missing', arg_group='Authentication')
 
+register_cli_argument('vmss', 'enable_autoscale', arg_group='Autoscale', action='store_true', help='Create with CPU-based autoscale settings. Assumed if one or more scaling arguments are supplied.')
+register_cli_argument('vmss', 'scale_out_cpu', arg_group='Autoscale', action=VmssAutoScaleEnableAction, help='CPU percentage at which which to scale out.')
+register_cli_argument('vmss', 'scale_out_increment', arg_group='Autoscale', action=VmssAutoScaleEnableAction, help='Number of VM instances to add when scaling out.')
+register_cli_argument('vmss', 'scale_out_max', arg_group='Autoscale', action=VmssAutoScaleEnableAction, help='Maximum number of VMs to provision.')
+register_cli_argument('vmss', 'scale_in_cpu', arg_group='Autoscale', action=VmssAutoScaleEnableAction, help='CPU percentage at which to scale in.')
+register_cli_argument('vmss', 'scale_in_increment', arg_group='Autoscale', action=VmssAutoScaleEnableAction, help='Number of VM instances to remove when scaling in.')
```

Comment:
Ignore this comment if it is a well-known terminology. To me, the name should be `scale_in_decrement`?